sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -1636,15 +1635,23 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     }
 
     withTable("rectangles", "rectangles2") {
```

Comment:
Mind changing `rectangles2` to `rectPartitioned` or similar? Think the test would get simpler to read (later).